crates/bevy_text/src/pipeline.rs

```diff
@@ -0,0 +1,120 @@
+use std::collections::HashMap;
+
+use ab_glyph::{PxScale, ScaleFont};
+use bevy_asset::{Assets, Handle, HandleId};
+use bevy_math::Size;
+use bevy_render::prelude::Texture;
+use bevy_sprite::TextureAtlas;
+
+use glyph_brush_layout::{FontId, SectionText};
+
+use crate::{
+    error::TextError, glyph_brush::GlyphBrush, Font, FontAtlasSet, TextAlignment, TextVertex,
+};
+
+pub struct TextPipeline {
+    pub brush: GlyphBrush,
```

Comment:
do these fields need to be public? otherwise its probably better to make them private to prevent accidental misuse.